tests/TestHarness/queries.py

```diff
@@ -625,8 +625,10 @@ def processUrllibRequest(self, resource, command, payload={}, silentErrors=False
             if Utils.Debug:
                 end=time.perf_counter()
                 Utils.Print("cmd Duration: %.3f sec" % (end-start))
-                printReturn=json.dumps(rtn) if returnType==ReturnType.json else rtn
-                Utils.Print("cmd returned: %s" % (printReturn[:1024]))
+                indent = 2 if prettyPrint else None
+                separators = (', ',': ') if prettyPrint else (',',':')
+                printReturn=json.dumps(rtn, indent=indent, separators=separators) if returnType==ReturnType.json else rtn
```

Comment:
I don't know how anyone else feels about this code or if it's idiomatic Python; however, I personally don't find it as clear as the if/elif/else equivalent:
```suggestion
    printReturn=None
    if returnType!=ReturnType.json:
        printReturn=rtn
    elif prettyPrint:
        printReturn=json.dumps(rtn, separators=(',',': '), indent=2 )
    else:
        printReturn=json.dumps(rtn, separators=(',',':'))
```

Something to consider, not a requirement.